index.d.ts

```diff
@@ -29,9 +29,13 @@ declare namespace JwksRsa {
     strictSsl?: boolean;
     requestHeaders?: Headers;
     timeout?: number;
+    getKeysInterceptor?(cb: (err: Error | null, keys: SigningKey[]) => void): void;
   }
 
   interface ClientOptionsWithObject extends Omit<ClientOptions, 'jwksUri'> {
+    /**
+     * @deprecated jwksObject should not be used. Use getKeysInterceptor as a replacement
```

Comment:
we don't need to deprecate if it never got released

Reply:
it got released 👎